tests/test_intf_mac.py

```diff
@@ -0,0 +1,248 @@
+from swsscommon import swsscommon
+
+import time
+import json
+
+class TestRouterInterfaceMac(object):
```

Comment:
will you support change the source MAC address for the router interfaces? if yes, could you add one more test case to change the source MAC address and validate the result?

Reply:
> please check the comments.
> 
> one more question, will you support remove the source MAC address once it is set?

@stcheng  Do you mean that set source mac address to default value? Unfortunately the existing producerstate class doesn't  support to del one attribute of the specific entry. You have to explicit set src mac address attribute of the INTERFACE_TABLE entry on appdb to default value(most likely system mac address).

Reply:
@shine4chen I agree. No need to remove the value.